README.md

```diff
@@ -88,16 +88,13 @@ docker pull pingcap/pd
 Run a single node with Docker: 
 
 ```bash
-# Set correct HostIP here. 
-export HostIP="192.168.199.105"
-
 docker run -d -p 2379:2379 -p 2380:2380 --name pd pingcap/pd \
           --name="pd" \
           --data-dir="pd" \
           --client-urls="http://0.0.0.0:2379" \
-          --advertise-client-urls="http://${HostIP}:2379" \
+          --advertise-client-urls="http://0.0.0.0:2379" \
```

Comment:
IMO, it is not good to assign 0.0.0.0 to the advertise url.

Reply:
Run a single node with Docker.

It should be 

```bash
docker run -d -p 2379:2379 -p 2380:2380 --name pd pingcap/pd \
          --name="pd" \
          --data-dir="pd" \
          --client-urls="http://0.0.0.0:2379" \
          --peer-urls="http://0.0.0.0:2380"
```

Reply:
@yangwenmai 

You can mount the PD port to your host port and set the host:mounted_port as the advertised URL.

Using 0.0.0.0 as the advertised URL is not recommended.

Reply:
😓
The reason I am wrong here is not that I did not mount the PD port to my host port, but my docker network and host communication problems.